.gitignore

```diff
@@ -30,3 +30,6 @@ compile_commands.json
 .ccls-cache/
 .mypy_cache
 .envrc
+clang_build
+build_profile
+build_profile.ninja
```

Comment:
unlike the other ignored files, these two files should be removed once the `optimized_clang.sh` finishes, so if everything works as expected, they should not stay with us, right?